mission/edit_me/briefing/objectives.html

```diff
@@ -0,0 +1,7 @@
+<font color='#E3D310'>PRIMARY OBJECTIVES:</font>
+
+INSERT OBJECTIVES THAT MUST BE COMPLETED TO ACHIEVE THE MISSION GOAL
+
+<font color='#E3D310'>SECONDARY OBJECTIVES:</font>
+
+INSERT OBJECTIVES THAT ARE NOT REQUIRED TO COMPLETE THE MISSION, BUT MAY BE USEFUL
```

Comment:
Not a fan of this OBJECTIVES segment - and it even feels redundant against MISSION, and I'm even less of a fan of there being primary and secondary objectives. I think we should let players be creative and insightful when trying to figure a mission out, this is too much hand-holding, maybe something similar could do, but not in the form of objectives. 

This is unnecessary bloat which players should figure out themselves after reading SITUATION and MISSION, making their own plans and pointers on how to take on the mission; we should not give them the solution to the mission from the get-go, people should be allowed to make mistakes if they don't read, or if they aren't attentive to other details of the MISSION and SITUATION, we should allow them to figure it out themselves, this is something that should happen in-game, not before.

Might make it easier for some people to take up leadership positions with all this sorted out, but this is setting the bar too low in my opinion.

Reply:
> this is too much hand-holding
> This is unnecessary bloat which players should figure out themselves
> not give them the solution

What would we be contracted for, if not for objectives?

This is not giving them the solution, this is not hand-holding or bloat, this is the whole point of the mission.

```
PRIMARY OBJECTIVE:
- Find and secure the stolen thing
```

Reply:
Players should be able to infer the tasks from reading their MISSION and interpreting what they should do in tandem with the SITUATION. Why does this not suffice? It's even borderline redundant.

This is gonna motivate some people to read the watered down tasks instead of the rest of the important information, and leaves aside the comprehension, internalization, and understanding what and why players are there. It may clear up what to do, but it distracts from understanding what is being done.

Players should read the MISSION, and then sort out how to achieve it and give themselves objectives; not be handed a step-by-step recipe on what to do; let them think for themselves, let them hear it from eachother.